filament/backend/src/vulkan/VulkanDescriptorSetCache.h

```diff
@@ -68,14 +70,20 @@ class VulkanDescriptorSetCache {
     fvkmemory::resource_ptr<VulkanDescriptorSet> createSet(Handle<HwDescriptorSet> handle,
             fvkmemory::resource_ptr<VulkanDescriptorSetLayout> layout);
 
+    // This method is meant to be used with external samplers
```

Comment:
Do these comments mean these methods are _only_ meant to be used with external samplers?

Reply:
for users outside of this class, yes.  Inside this class, I do use this function (`getVkSet`) even in the general case.

Reply:
Gotcha. Maybe clarify in the comment.

Reply:
done